python/action_server/stateMachine.py

```diff
@@ -0,0 +1,195 @@
+from transitions.extensions.markup import MarkupMachine
+from transitions.extensions.factory import HierarchicalMachine
+from transitions import MachineError
+import json
+from config import ZenohConfig
+import zenoh
+import logging
+
+logging.getLogger().setLevel(logging.DEBUG)
+
+"""
+Global variables:
+    QUEUED(bool): To process the tranisitions in a queue.
+    args: object of the ZenohConfig class which validates zenoh configuration variables.
+    conf: Zenoh configuration object from zenohd.
+    session: object to create a session.
+    pub: object for publisher to publish executed state on zenohd.
+"""
+conf, args = ZenohConfig().zenohconfig()
+zenoh.init_logger()
+session = zenoh.open(conf)
+pub = session.declare_publisher(args.base_key_expr + "/state")
```

Comment:
very bad practice, globle variables are a bad practice

Reply:
Binded this code under a function.